worker/Cargo.toml

```diff
@@ -5,3 +5,4 @@ edition.workspace = true
 lints.workspace = true
 
 [dependencies]
+sysinfo = "0.30.7"
```

Comment:
Use a workspace dependency instead so that all versions are centrally specified. [Learn more here.](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table)

Reply:
When importing a unpublished crate (e.g. proto), is there an appropriate convention? I´ve seen the one used in the Cargo book [here](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-path-dependencies), is a more concise way of doing it?

Reply:
Yes, after #5 is merged you'll be able to do `proto.workspace = true` on the `worker`'s Cargo.toml manifest.